frontend/src/components/login/register.js

```diff
@@ -13,32 +20,48 @@ export class Register extends React.Component {
+                        <input type="text" name="username" placeholder="username" onChange = {e => this.setState({username: e.target.value})}/>
                     </div>
                     <div className="form-group">
                         <label htmlFor="f_name">First name</label>
-                        <input type="text" name="first name" placeholder="first name"/>
+                        <input type="text" name="first name" placeholder="first name" onChange = {e => this.setState({fName: e.target.value})}/>
                     </div>
                     <div className="form-group">
                         <label htmlFor="l_name">Last name</label>
-                        <input type="text" name="last name" placeholder="last name"/>
+                        <input type="text" name="last name" placeholder="last name" onChange = {e => this.setState({lname: e.target.value})}/>
                     </div>
                     <div className="form-group">
                         <label htmlFor="email">Email</label>
-                        <input type="text" name="email" placeholder="email"/>
+                        <input type="text" name="email" placeholder="email" onChange = {e => this.setState({email: e.target.value})}/>
                     </div>
                     <div className="form-group">
                         <label htmlFor="password">Password</label>
-                        <input type="password" name="password" placeholder="password"/>
+                        <input type="password" name="password" placeholder="password" onChange = {e => this.setState({pass: e.target.value})}/>
                     </div>
                     <div className="form-group">
                         <label htmlFor="verify_password">Verify Password</label>
-                        <input type="password" name="v_password" placeholder="verify password"/>
+                        <input type="password" name="v_password" placeholder="verify password" onChange = {e => this.setState({verify_password: e.target.value})}/>
                     </div>
                 </div>
             </div>
             <div className="footer">
-                <button type="button" className="btn">
+                <button type="button" className="btn"
+                onClick={async () => {
+                  if (this.verify_password == this.pass) {
+                    fetch('/auth/signup', {
+                      method: 'POST',
+                      body: JSON.stringify({
+                        "first_name": this.state.fName,
+                        "last_name":this.state.lname,
+                        "email": this.state.email,
+                        "username": this.state.username,
+                        "password": this.state.pass,
+                        "verify_password": this.state.verify_password
+                      })
+                    } )
+                   }
+                   console.log(this.state)
```

Comment:
The server response also needs to be checked to see if account creation was successful cuz I'm not seeing any error messages when I incorrectly fill out the fields.